test/src/tests-consent.js

```diff
@@ -3,19 +3,14 @@ import sinon from 'sinon';
 import { urls } from './config';
 import { apiKey, MPConfig, testMPID } from './config';
 
-var getEvent = Utils.getEvent,
+var findBatch = Utils.findBatch,
```

Comment:
Nitpick: Can we rename `test/src/utils.js` to `test/src/test-utils.js`? I even think the object should be renamed to `TestUtils` for clarity, but that's more of a personal preference.

Reply:
My main reason for this is so that we don't confuse `Utils` for `TestUtils`.